DiffPlex.Wpf/Controls/Helper.cs

```diff
@@ -41,15 +41,26 @@ internal static class Helper
     internal static void RenderInlineDiffs(InternalLinesViewer viewer, ICollection<DiffPiece> lines, UIElement source, int contextLineCount)
     {
         viewer.Clear();
+        var diffViewer = source as DiffViewer;
```

Comment:
Shouldn't this work also for InlineDiffviewer too? If so, we can make a common interface for both to pass into this instead of UIElement.

Reply:
Making a common interface would mean that `Helper.cs` would have to implement all the methods from InternalLinesViewer as well. I will continue using safe casting for both `InternalLinesViewer` and `Helper.cs`

Reply:
This has been pushed